jaeger-tracerresolver/build.gradle

```diff
@@ -0,0 +1,34 @@
+description = 'TracerResolver implementation for Jaeger Tracer'
+
+dependencies {
+    compile project(':jaeger-core')
+    compile group: 'io.opentracing.contrib', name: 'opentracing-tracerresolver', version: tracerResolverVersion
+ 
+    testCompile group: 'junit', name: 'junit', version: junitVersion
+
+    signature 'org.codehaus.mojo.signature:java16:1.1@signature'
+}
+
+sourceSets {
```

Comment:
The previous maybe yes. This can be done independently it does not affect other modules.